lib/saml/user_attributes/ssoe.rb

```diff
@@ -197,6 +198,17 @@ def validate!
 
       private
 
+      def should_raise_idme_uuid_error
```

Comment:
I suggest we clean up the logic here a bit, make it look more 'rubyish':

> def should_raise_idme_uuid_error
      return false unless idme_uuid
    @saml_settings.nil? ||
    @saml_settings.assertion_consumer_service_url =~ %r{/v1/} && (@authn_context != INBOUND_AUTHN_CONTEXT)
end